src/backend/common/models/alliance.py

```diff
@@ -10,6 +11,7 @@ class _EventAllianceOptional(TypedDict, total=False):
     declines: List[TeamKey]
     name: str
     backup: EventAllianceBackup
+    status: EventTeamStatusPlayoff
```

Comment:
Is this necessary? I don't see where it is being used.

Reply:
This is a change to make the python model match what's in the DB. Without it, the type check fails: https://github.com/the-blue-alliance/the-blue-alliance/actions/runs/9097667894/job/25006007976

This seems to be the "intended" material of the alliance data, as even the test data includes it (see https://github.com/the-blue-alliance/the-blue-alliance/blob/py3/src/backend/common/helpers/tests/data/2024test_alliances.json and other backend helper data), but I can't for the life of me figure out where the data is coming from.

Reply:
Oh i see. yeah this must've been overlooked originally somehow.